streaming/python/tests/test_hybrid_stream.py

```diff
@@ -57,13 +57,23 @@ def sink_func(x):
         .sink(sink_func)
     ctx.submit("HybridStreamTest")
     import time
-    time.sleep(3)
+    slept_time = 0
+    while True:
+        if os.path.exists(sink_file):
+            time.sleep(3)
+            with open(sink_file, "r") as f:
+                result = f.read()
+                assert "a" in result
+                assert "b" not in result
+                assert "c" in result
+            print("Execution succeed")
+            break
+        if slept_time >= 60:
+            raise Exception("Execution not finished")
```

Comment:
To implement this pattern, we can use `ray.test_utils.wait_for_condition`

Reply:
Good suggestion